### PR TITLE
cla-escodoo

### PR DESCRIPTION
**Pull request by marcelsavegnago**
Description of the issue/feature this PR addresses:

Include CLA Corporate - Escodoo

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/cla/corporate/escodoo.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+I declare that I am authorized and able to make this agreement and sign this declaration.
+
+Signed,
+
+Marcel Savegnago marcel.savegnago@escodoo.com.br https://github.com/escodoo
+
+List of contributors:
+
+Marcel Savegnago marcel.savegnago@escodoo.com.br https://github.com/marcelsavegnago Eduardo Aparício eduardo.aparicio@escodoo.com.br https://github.com/eduaparicio
```
Comment (mart-e) [inline doc/cla/corporate/escodoo.md:13]: maybe put it with one person per line

**Review by mart-e — COMMENTED** [review]
Hi !

Thanks for the CLA signature. 
Could you change the commit to follow the [commit message guidelines](https://www.odoo.com/documentation/12.0/reference/guidelines.html#commit-message-structure), to something like
```
[CLA] Escodoo signs Odoo's CCLA
```

While this is not an issue to sign the CLA and contribute to Odoo, please note that our [partnership agreement](https://www.odoo.com/documentation/user/13.0/legal/terms/partnership.html#brand-image) forbids to use the name "ODOO" in their company name. If you consider to become an official partner one, you may want to rebrand.

**Comment by marcelsavegnago** [comment]
> While this is not an issue to sign the CLA and contribute to Odoo, please note that our [partnership agreement](https://www.odoo.com/documentation/user/13.0/legal/terms/partnership.html#brand-image) forbids to use the name "ODOO" in their company name. If you consider to become an official partner one, you may want to rebrand.

About the use of the name I had no problems when we made the partnership. Currently, we no longer have a partnership with Odoo S.A.
Thank you for the information and we will double check the partnership agreement.

**Comment by mart-e** [comment]
Thanks for the update.

The naming part was added recently (odoo/documentation-user@aa53cf0688835b32f679df708fd1c34be759b681) to the contracts. So if you have signed it before that, it was not a rule back them :)

For the CLA, looks good now, it will soon be integrated into Odoo.
Future pull requests made on odoo will be validated by the runbot.
If you have existing PRs that are awaiting for CLA validation, don't forget to rebase your code as soon as this is merged.

Regards

@robodoo review+

**Comment by mart-e** [comment]
@robodoo r-

Sorry, checked too fast. Actually, have left 4 commits. Could you rebase and squash to keep only one with the last commit message ?
Thanks

**Comment by marcelsavegnago** [comment]
> @robodoo r-
> 
> Sorry, checked too fast. Actually, have left 4 commits. Could you rebase and squash to keep only one with the last commit message?
> Thanks

Done

**Comment by mart-e** [comment]
Thanks !
@robodoo r+ again !

**Pull request closed without merging**